policies/policies.go

```diff
@@ -29,26 +29,92 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	"os"
+	"strings"
 
 	"clouditor.io/clouditor/api/evidence"
 	"github.com/open-policy-agent/opa/rego"
 )
 
-func RunEvidence(file string, evidence *evidence.Evidence) (data map[string]interface{}, err error) {
+// applicableMetrics stores a list of applicable metrics per resourceType
+var applicableMetrics = make(map[string][]string)
+
+func RunEvidence(evidence *evidence.Evidence) ([]map[string]interface{}, error) {
+	data := make([]map[string]interface{}, 0)
+	var baseDir string = "."
+	// check, if we are in the root of Clouditor
+	if _, err := os.Stat("policies"); os.IsNotExist(err) {
+		// in tests, we are relative to our current package
+		baseDir = ".."
+	}
+
 	var m = evidence.Resource.GetStructValue().AsMap()
 
-	return RunMap(file, m)
+	var types []string
+
+	if rawTypes, ok := m["type"].([]interface{}); ok {
+		types = make([]string, len(rawTypes))
+	} else {
+		return nil, fmt.Errorf("got type '%T' but wanted '[]interface {}'. Check if resource types are specified ", rawTypes)
+	}
+	for i, v := range m["type"].([]interface{}) {
+		// TODO(all): type assertion check good or unnecessary because we assume resourceTypes to be always set as intended ([]string)?
```

Comment:
type assertions should always be done in a "safe" way, e.g., using the `newValue, ok := oldValue.(string)` pattern, otherwise the program might crash.